backend/app/core/config.py

```diff
@@ -59,5 +59,5 @@
     "OPENSEARCH_INDEX_TEXT_BLOCK_KEY", "text_block_id"
 )
 OPENSEARCH_INDEX_ENCODER: str = os.getenv(
-    "OPENSEARCH_INDEX_ENCODER", "msmarco-distilbert-dot-v5"
+    "OPENSEARCH_INDEX_ENCODER", "sentence-transformers/msmarco-distilbert-dot-v5"
```

Comment:
Wondering if we need to bring `OPENSEARCH_INDEX_ENCODER` in here somehow and in the Dockerfile, but then should it really be configurable in the first place? E.g. Could it ever be set to a value that's incompatible with the code?